fvtest/compilertriltest/ShiftAndRotateTest.cpp

```diff
@@ -195,17 +268,57 @@ TEST_P(Int8ShiftAndRotate, UsingConst) {
     ASSERT_EQ(param.oracle(param.lhs, param.rhs), entry_point());
 }
 
+TEST_P(Int8ShiftAndRotate, UsingRhsConst) {
+    auto param = TRTest::to_struct(GetParam());
+
+    std::string arch = omrsysinfo_get_CPU_architecture();
+    SKIP_IF(OMRPORT_ARCH_S390 == arch || OMRPORT_ARCH_S390X == arch, KnownBug)
+        << "The Z code generator incorrectly spills sub-integer types arguments (see issue #3525)";
+
+    char inputTrees[120] = {0};
+    std::snprintf(inputTrees, 120, "(method return=Int8 args=[Int8] (block (ireturn (b2i (%s (bload parm=0) (iconst %d))))))", param.opcode.c_str(), param.rhs);
```

Comment:
Can we format the Tril test strings in a tree like form similar to how our IL looks? It makes debugging and reading the trees much easier. See [1] for example.

[1] https://github.com/eclipse/omr/blob/14248fb2342a51067ce38921f34aeda8dfb7cc53/fvtest/compilertriltest/IfxcmpgeReductionTest.cpp#L97-L114

Reply:
Fixed in d13237aa5cdddb4c201d53296d4cfdd9b57c45ea